robottelo/ui/oscappolicy.py

```diff
@@ -9,8 +9,16 @@
 class OpenScapPolicy(Base):
     """Manipulates OpenScap Policy from UI"""
 
+    def navigate_to_entity(self):
+        """Navigate to OpenScap Policy entity page"""
+        Navigator(self.browser).go_to_oscap_policy()
+
+    def _search_locator(self):
+        """Specify locator for OpenScap Policy entity search procedure"""
+        return locators['oscap.select_policy']
```

Comment:
`common_locators['search']` was used before. Is it specific reason for that new locator?